cmd/build/helmify/kustomize-for-helm.yaml

```diff
@@ -76,7 +76,7 @@ spec:
             - --logtostderr
             - --log-denies={{ .Values.logDenies }}
             - --emit-admission-events={{ .Values.emitAdmissionEvents }}
-            - --log-level={{ .Values.logLevel }}
+            - --log-level={{ (.Values.controllerManager.logLevel | empty | not) | ternary .Values.controllerManager.logLevel .Values.logLevel }}
```

Comment:
is this backwards compatible?

Reply:
Yes, if these new values aren't specified, these manifests will still get the value from `.Values.logLevel`.

![image](https://user-images.githubusercontent.com/47261237/200862103-33c97b68-57cf-48dd-8ff4-34b7be074d8f.png)